doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -224,25 +227,27 @@ int main(int argc, char **argv)
 
   // We want the cartesian path to be interpolated at a resolution of 1 cm
   // which is why we will specify 0.01 as the max step in cartesian
-  // translation.  We will specify the jump threshold as 0.0, effectively
-  // disabling it.
+  // translation.  We will specify the jump threshold as 0.0, effectively disabling it.
```

Comment:
I agree with you - i would never disable the jump threshold. but i don't want to get into that here. ive added a comment on safety